apps/indexer/src/api/mappers/proposals.ts

```diff
@@ -46,7 +46,9 @@ export const ProposalResponseSchema = z.object({
   forVotes: z.string(),
   againstVotes: z.string(),
   abstainVotes: z.string(),
+  startTimestamp: z.string(),
```

Comment:
Is startTimestamp != timestamp ?

timestamp is the moment that the prop was submitted and startTimestamp is the moment that the prop wil be active?